feature/about/src/main/java/io/github/droidkaigi/confsched2019/about/ui/item/AboutItem.kt

```diff
@@ -16,12 +17,23 @@ class AboutItem(
     override fun getLayout(): Int = R.layout.item_about
 
     override fun bind(binding: ItemAboutBinding, position: Int) {
+        // clickListener is null when this item is version item
+        val isVersionItem = clickListener == null
```

Comment:
When adding item, if that person do not know this spec, it will cause bug.
Please add isLaketonCheckText field to AboutItem 🙏

Reply:
@takahirom 
Thanks reply.
Please wait as it fixes. 🙇